Dockerfile

```diff
@@ -0,0 +1,8 @@
+### STAGE 1: Build ###
+FROM node:18.14.0 AS build
+WORKDIR /usr/src/app
+COPY . .
+RUN npm install --ignore-scripts
+
+### STAGE 2: Run ###
+CMD ["npm", "run", "start"]
```

Comment:
Create a Dockerfile following this instruction
https://dev.to/aws-builders/containerize-react-app-with-docker-for-production-572b

To learn more, you can also follow this video:
https://dev.to/aws-builders/containerize-react-app-with-docker-for-production-572b